Dart/resources/messages/DartBundle.properties

```diff
@@ -226,6 +226,8 @@ warning.no.projects.selected.dart.support.will.be.disabled=Warning\: no projects
 debugger.trying.to.connect=Trying to connect
 enter.url.to.running.dart.app=Enter a URL to a running Dart or Flutter application
 connect.to.running.app.title=Connect to a Running App
+connect.to.running.app.comment=Running Dart processes have a VM service that can be connected to via a URL, the URL to connect to the VM \
```

Comment:
I'd avoid mentioning the Observatory specifically.

I believe the text is a run-on sentence; I'd re-work into multiple sentence, use a semi-colon, or similar.